src/otx/engine/utils/auto_configurator.py

```diff
@@ -274,6 +286,11 @@ def get_model(self, model_name: str | None = None, label_info: LabelInfoTypes |
 
         model_config = deepcopy(self.config["model"])
 
+        if input_size is not None:
+            model_config["init_args"]["input_size"] = (
+                (input_size, input_size) if isinstance(input_size, int) else input_size
+            )
```

Comment:
Are we safe to forward mean/scale form the config?
Related to https://github.com/openvinotoolkit/training_extensions/issues/3827

Reply:
yes I think. but currently, there are no arguments for model init function. We need to refactor model code to remove hard coded mean and std. It's already discussed so, we can refactor that part later.